tox.ini

```diff
@@ -53,9 +53,10 @@ deps=
     mrcfile
     pyxem
     pims
+    scikit-image<0.19
```

Comment:
Since it is an issue within `pyxem`, I'm hoping there will be a fix released at some time. Since it is only used for testing, we can maybe skip these tests for Python 3.10 until `pyxem` is fixed?

Reply:
Sounds good - 3.10 support is still waiting on a few releases anyway (most importantly numba, but primesieve is also currently built from source on 3.10, which would be painful on Windows...)

Reply:
See also https://github.com/LiberTEM/LiberTEM/issues/1133